core/src/test/scala/org/apache/spark/benchmark/Benchmark.scala

```diff
@@ -200,11 +200,12 @@ private[spark] object Benchmark {
   def getProcessorName(): String = {
     val cpu = if (SystemUtils.IS_OS_MAC_OSX) {
       Utils.executeAndGetOutput(Seq("/usr/sbin/sysctl", "-n", "machdep.cpu.brand_string"))
+        .stripLineEnd
```

Comment:
Because the Mac has one more line than Linux:
https://github.com/apache/spark/pull/22661/commits/28f9b9a8a26caf8750aa2e8c8e2bc793b3773d98#diff-45c96c65f7c46bc2d84843a7cb92f22fL7

Reply:
Ur.. I'm not a fan to piggy-backing. Okay.